shared/fs/nav-header/title.tsx

```diff
@@ -84,10 +84,29 @@ const Breadcrumb = Kb.OverlayParentHOC(
   ))
 )
 
+const MaybePublicTag = (props: Props) => {
+  const parsedPath = Constants.parsePath(props.path)
+  switch (parsedPath.kind) {
+    case 'group-tlf':
+    case 'in-group-tlf':
+      if (parsedPath.tlfType === 'public') {
+        return (
+          <Kb.Box2 direction="horizontal">
+            <Kb.Meta title="public" backgroundColor={Styles.globalColors.green} />
+          </Kb.Box2>
+        )
+      }
+      break
+    default:
+  }
+  return null
+}
```

Comment:
Same here. If you prefer to use `showPublicTag` above, it should probably go into `constants/fs` so it can be used here too.